xil_carma_messenger/carma-messenger-docker.launch

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
```

Comment:
we don't need this file since we are only launching messenger_ros2.
And we will add the ros2bag in the docker.launch.py as we discussed offline

Reply:
sorry I meant carma-messenger.launch.py*